src/item_menu.c

```diff
@@ -1,3 +1,4 @@
+#include <string.h>
```

Comment:
Because `#include "string.h"` was making scaninc look in our include dirs for a file named `string.h`. It's more correct to specify that it's a library header, not a local one.